onnxruntime/core/graph/function.cc

```diff
@@ -274,8 +274,9 @@ FunctionImpl::FunctionImpl(const onnxruntime::Graph& graph,
     onnxruntime::NodeAttributes new_attr_map;
     for (auto& attr : node.attribute()) {
       if (attr.has_ref_attr_name()) {
-        if (attr_map.count(attr.ref_attr_name())) {
-          new_attr_map[attr.name()] = attr_map[attr.ref_attr_name()];
+        auto entry = attr_map.find(attr.ref_attr_name());
```

Comment:
curious: isn't count() achieving the same thing? (i wish stl had 'contains')

Reply:
In general, prefer 'find' over 'count' as the former will stop once it finds a match. Depending on the container type that can be a significant different. 

Additionally in this case we want to use the entry, so 'count' + operator[] == two lookups vs one for using the iterator returned by 'find'.